src/apolo_app_types/protocols/job.py

```diff
@@ -483,8 +456,18 @@ class JobAppInput(AppInputs):
         default_factory=JobNetworkingConfig,
     )
 
-    integrations: JobIntegrationsConfig = Field(
-        default_factory=JobIntegrationsConfig,
+    mlflow_integration: MLFlowTrackingServerURL = Field(
```

Comment:
can you please clarify it? since the idea was to expand a list of integrations and hide them under `advanced` collapsable element